src/compiler/transformers/generators.ts

```diff
@@ -1124,6 +1124,18 @@ namespace ts {
                 //  .mark resumeLabel
                 //      _b.apply(_a, _c.concat([%sent%, 2]));
 
+                // The `ImportCall` Expression isn't really a call expression, and the import keyword cannot be removed from its arguments

```

Comment:
The generators transform was written to expect only ES5-compatible code in a function body. However, we moved the module transformer to the end of the transformation pipeline, which is where `import` is handled. I'm ok with this change, though I wonder what the effect of moving generators (but not the es2015 transform) after the module transformer...